server/mysql_storage_provider.go

```diff
@@ -30,6 +30,7 @@ import (
 
 var (
 	mySQLURI = flag.String("mysql_uri", "test:zaphod@tcp(127.0.0.1:3306)/test", "Connection URI for MySQL database")
+	maxConns = flag.Int("max_db_conns", 1000, "Maximum connections to the database")
```

Comment:
The default ought to be 0 so that this doesn't affect anyone not using the flag. We can consider setting a non-zero default value later, once it's been tested out for a while.

Reply:
Also, it'd be nicer if this was named "mysql_max_conns". The "mysql_" prefix is so that it clearly relates to the MySQL storage implementation.

Reply:
Done! Sorry for so many rounds of review, I should have been paying closer attention. :-D